libcnb-test/src/lib.rs

```diff
@@ -85,15 +86,37 @@ impl IntegrationTest {
     /// - When the connection to Docker failed
     /// - When the internal Tokio runtime could not be created
     pub fn new(builder_name: impl Into<String>, app_dir: impl AsRef<Path>) -> Self {
+        let tokio_runtime =
+            tokio::runtime::Runtime::new().expect("Could not create internal Tokio runtime");
+
+        let docker = match env::var("DOCKER_HOST") {
+            #[cfg(target_family = "unix")]
+            Ok(docker_host) if docker_host.starts_with("unix://") => {
+                Docker::connect_with_unix_defaults()
+            }
+            Ok(docker_host)
+                if docker_host.starts_with("tcp://") || docker_host.starts_with("https://") =>
+            {
+                Docker::connect_with_ssl_defaults()
+            }
+            Ok(docker_host) => panic!(
+                "Cannot connect to unsupported DOCKER_HOST '{}'",
+                docker_host
+            ),
+            Err(VarError::NotPresent) => Docker::connect_with_local_defaults(),
+            Err(VarError::NotUnicode(_)) => {
+                panic!("DOCKER_HOST environment variable is not unicode encoded!")
```

Comment:
(pedantic) I suppose the cause of this error could also be that it's unicode encoded, just not correctly? The [NotUnicode](https://doc.rust-lang.org/std/env/enum.VarError.html#variant.NotUnicode) docs use the phrase "did not contain valid unicode".

Reply:
Is it unicode if it's not valid unicode? ;-)